anaconda.py

```diff
@@ -580,19 +580,8 @@ def _earlyExceptionHandler(ty, value, traceback):
     ksdata.displaymode.displayMode = display_mode_coversion_table[anaconda.display_mode]
     ksdata.displaymode.nonInteractive = not anaconda.interactive_mode
 
-    # If we're in text mode, the resulting system should be too
-    # ...unless the kickstart specified otherwise
-    # NOTE: Installation controlled via VNC is considered to be
-    #       a text mode installation, as the installation run itself
-    #       is effectively headless.
-    from pyanaconda.modules.common.constants.services import SERVICES
-    from pyanaconda.core.constants import TEXT_ONLY_TARGET
-
-    services_proxy = SERVICES.get_proxy()
-
-    if not services_proxy.DefaultTarget and (anaconda.tui_mode or flags.usevnc):
-        log.debug("no default systemd target set & in text/vnc mode - setting multi-user.target.")
-        services_proxy.SetDefaultTarget(TEXT_ONLY_TARGET)
+    # Set up the default target.
+    startup_utils.initialize_default_target(text_mode=anaconda.tui_mode)
```

Comment:
This should have better name. What target? It's fine here but not really self-explanation in the startup_utils with all the other stuff around. Please rename this to something like `initialize_systemd_default_target`?

Reply:
Fixed.